tests/Unit/Contrib/JaegerSpanConverterTest.php

```diff
@@ -26,7 +26,7 @@ public function test_should_convert_an_otel_span_to_a_jaeger_thrift_span()
         $span = (new SpanData())
                     ->setName('otelSpanName');
 
-        $jtSpan = (new SpanConverter())->convert($span);
+        [$jtSpan] = (new SpanConverter())->convert([$span]);
```

Comment:
What does `jtSpan`mean?

Reply:
I think the agent exporter was using it as shorthand for "Jaeger Thift Span" iirc, also in part because the thrift library's exported type was also called "Span", so I think in the converter file this was renamed to JTSpan

Definitely open to switching it up though however is more understandable

Reply:
@grunet
My comment was more about readabilty. I believe it should be instantly understandable what a variable (or anything else)
means and that's often hard enough with English being such an ambiguous language (at least compared to German).
The point is, code is more often read than written, so anything that helps to easily understand the code is useful imo, even if it's often cumbersome to write something like `$value` instead of just `$v`

Reply:
Yeah for sure. "convertedSpan" looked good to me so I went ahead and switched that in